packages/core/src/browser/view-container.ts

```diff
@@ -369,11 +369,6 @@ export class ViewContainer extends BaseWidget implements StatefulWidget, Applica
         return { parts: partStates, title: this.titleOptions };
     }
 
-    /**
-     * The view container restores the visibility, order and relative sizes of contained
-     * widgets, but _not_ the widgets themselves. In case the set of widgets is not fixed,
-     * it should be restored in the specific subclass or in the widget holding the view container.
-     */
```

Comment:
This comment is outdated. @akosyakov implemented restoring of contained widgets in https://github.com/eclipse-theia/theia/commit/93205109fa4017286ec0a441b0250b74b36c6495

Reply:
#ok